ketcher-autotests/tests/Structure-Creating-&-Editing/Atom/Charge-Tool/charge-tool.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  openFileAndAddToCanvas,
+  takeLeftToolbarScreenshot,
+  selectRingButton,
+  RingButton,
+  moveOnAtom,
+  clickInTheMiddleOfTheScreen,
+  resetCurrentTool,
+  selectLeftPanelButton,
+  LeftPanelButton,
+  clickOnAtom,
+} from '@utils';
+
+test.describe('Charge tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Hot key for the Charge Plus Tools', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1664
+    Description: Charge Plus is applied to the structure atom.
+    */
+    const anyAtom = 0;
+    await selectRingButton(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await moveOnAtom(page, 'C', anyAtom);
+    await page.keyboard.press('Shift++');
+    await resetCurrentTool(page);
+  });
+
+  test('Hot key for the Charge Minus Tools', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1664
+    Description: Charge Minus is applied to the structure atom.
+    */
+    const anyAtom = 0;
+    await selectRingButton(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await moveOnAtom(page, 'C', anyAtom);
+    await page.keyboard.press('-');
+    await resetCurrentTool(page);
+  });
+
+  test('Change charge on different atoms', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1665
+    Description: Charge Minus and Charge Plus is applied to the structure atom.
+    */
+    const anyAtom = 0;
+    const anyAtom2 = 2;
```

Comment:
Perhaps this constant should be renamed to "atomIndex" and instead of "anyAtom2" -> "anotherAtomIndex"